cairo/tests/ethereum/cancun/test_transactions.py

```diff
@@ -23,3 +33,62 @@ def test_validate_transaction(self, cairo_run_py, tx: Transaction):
             return
 
         assert result_cairo == validate_transaction(tx)
+
+    @given(tx=...)
+    def test_signing_hash_pre155(self, cairo_run, tx: LegacyTransaction):
+        """Test pre-EIP155 transaction signing hash computation"""
+        try:
+            cairo_result = cairo_run("signing_hash_pre155", tx)
+        except Exception as cairo_error:
+            with strict_raises(type(cairo_error)):
+                signing_hash_pre155(tx)
+            return
```

Comment:
are you expecting failures in these tests?

Reply:
ah thanks, I don't think so